### PR TITLE
Add class horse

### PR DESCRIPTION
**Pull request by tuanbui0509**
please check ...

### CODE REVIEW

File: horse.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function horse(name) {
+    this.name = name;
+}
+
```
Comment (buingoctuan99) [inline horse.js]: remove line break

**Review by buingoctuan99 — COMMENTED** [review]
nothing

File: horse.js (hunk 1 above)
Comment (tuanbui0509) [reply horse.js]: okie man

**Review by tuanbui0509 — COMMENTED** [review]
nothing

File: horse.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+function horse(name) {
+    this.name = name;
+
```
Comment (buingoctuan99) [inline horse.js]: please remove line break this line

**Review by buingoctuan99 — COMMENTED** [review]
haizzzzzzzzz

File: horse.js (hunk 2 above)
Comment (tuanbui0509) [reply horse.js]: okie main let me remove this line

**Review by tuanbui0509 — COMMENTED** [review]
:((

**Pull request merged**